src/ppc/src/compile.rs

```diff
@@ -154,6 +160,9 @@ impl<'a> ProblemBuilder<'a> {
         let mut out = vec![];
         for (i, test_spec) in self.cfg.tests.iter().enumerate() {
             let tid = i + 1;
+            if tid % 10 == 0 {
```

Comment:
Why 10? Why not 5?

Reply:
Well, it was working solution.
Better plan: print current test every 10 seconds.

Reply:
I think that'll be fine